client/db/bolt/db.go

```diff
@@ -622,6 +622,25 @@ func (db *BoltDB) DEXOrdersWithActiveMatches(dex string) ([]order.OrderID, error
 			if status[0] == uint8(order.MatchComplete) {
 				return nil
 			}
+
+			// Exclude refunded swaps.
+			// proofB := getCopy(mBkt, proofKey)
+			// if len(proofB) == 0 {
+			// 	log.Errorf("empty match proof")
+			// 	return nil
+			// }
+			// proof, errM := dexdb.DecodeMatchProof(proofB)
+			// if errM != nil {
+			// 	log.Errorf("error decoding proof: %v", errM)
+			// 	return nil
+			// }
+			// if len(proof.RefundCoin) > 0 {
+			// 	return nil
+			// }
+
+			// TODO: Could also filter out certain revoked matches depending on
+			// status. See (*trackedTrade).isActive.
```

Comment:
I saw the other comment, but I do think this would be fine as long as we're specific in the docs about what is being excluded.

Reply:
If we're excluding here, would we also need to exclude in `core.loadDBTrades`? Would be better to have this filtering done in one place so that future modifications do not require making similar changes to 2 portions of code.

Would also be pretty much irrelevant to filter here AND again in `core.loadDBTrades`. I agree with the thinking to keep db stuff high level and let core (and any other consumer) deal with the nitty-gritties.

Reply:
The methods are called `ActiveDEXOrders` and `DEXOrdersWithActiveMatches`. Refunded trades are not "active", so the DB is really wrong as it is now.

Reply:
Point taken.  I'll make the db method do it all, but the logic must be mirrored in isActive for retiring trades that complete

Reply:
This is done now.